$MirOS: ports/security/libgpg-error/patches/patch-aclocal_m4,v 1.1 2006/09/13 20:54:58 tg Exp $
--- aclocal.m4.orig	Thu Apr  3 20:12:40 2008
+++ aclocal.m4	Thu Apr  3 20:12:51 2008
@@ -930,7 +930,6 @@ m4_include([m4/iconv.m4])
 m4_include([m4/lib-ld.m4])
 m4_include([m4/lib-link.m4])
 m4_include([m4/lib-prefix.m4])
-m4_include([m4/libtool.m4])
 m4_include([m4/nls.m4])
 m4_include([m4/po.m4])
 m4_include([m4/progtest.m4])
